### DISCUSSION TITLE
does make_before_break work with Cisco ASA?

### DISCUSSION THREAD

**Discussion by felartu** [strongswan/strongswan#1260] [General]
Hello,

We have make_before_break = 1 in our strongswan.conf configuration but we are seeing strange reauth attempts from the other peer which is a Cisco ASA, we are wondering if make_before_break is supported by Cisco ASA and if so what is the name of the configuration?

If of any use we have:
uniqueids=replace
auto=route
ikelifetime=10m
lifetime=2m
margintime=5m

**Comment by tobiasbrunner** [comment]
That option only has a local effect i.e. whenever strongSwan is initiating a reauthentication itself. It doesn't influence when or how the other peer does reauthentications.
